lib/nylas/types.rb

```diff
@@ -79,5 +79,14 @@ def cast(value)
       end
     end
     Types.registry[:string] = StringType.new
+
+    # Type for attributes represented as booleans.
+    class BooleanType < ValueType
+      # @param value [Object] Strictly casts the passed in value to a boolean (must be true, not "" or 1)
+      def cast(value)
```

Comment:
I am wondering if this is nil, it should be nil?

That way if we get a nil value in something that could be boolean, we don't assume it's false. Thoughts @khamidou?

Reply:
Hmm, I don't think of a case where this would happen. Maybe we should raise an exception?

Reply:
I can see it in more generic use cases, but not with your API as your API doesn't send non-true-values. Exception makes sense!